Added few faqs (#6531)
harness/developer-hub: docs/troubleshooting/troubleshooting-nextgen.md

This feature is not available for Harness Self-Managed Enterprise Edition.

:::

### Connector not found while running the pipeline

> Error: Stage Stage_Name: Connector not found - [Connector_ID] with scope [Project]

Reviewer: ```suggestion
"Error: Stage Stage_Name: Connector not found - [Connector_ID] with scope [Project]"
```

> For these errors, please review whether the service has been configured with the correct artifact type connector or whether the specific connector type exists in the Project.

Reviewer: ```suggestion
For these errors, review whether the service is configured with the correct artifact type connector or whether the specific connector type exists in the project.
```
